nixos/modules/services/web-apps/bookstack.nix

```diff
@@ -385,13 +385,13 @@ in {
                 else if isString v then v
                 else if true  == v then "true"
                 else if false == v then "false"
-                else if isSecret v then v._secret
+                else if isSecret v then "@_@${v._secret}@_@"
```

Comment:
how about?
```suggestion
                else if isSecret v then "${placeholder v._secret}"
```

Reply:
Yeah, I was actually thinking of using `hashString`, which seems to work well, so I'll switch to that instead.